### PR TITLE
patch user: change primary email

### PR DESCRIPTION
**Pull request by kat-hi**

### CODE REVIEW

File: src/api/users.js [hunk 1]
```diff
@@ -11,28 +11,30 @@ const users_container = {
...
-      set_access_token (state, token) { state.access_token = token }
+      set_access_token (state, token) { state.access_token = token },
+      set_user_groups (state, user_groups) { state.user_groups = user_groups }
     },
 
     actions: {
-      async get_user_details (context, field) {
-        const response = await backend.readByField('/users', field)
+      async get_user_details (context, id) {
+        const response = await backend.readByField('/users', id)
```
Comment (troeger) [inline src/api/users.js:37]: Just a personal opinion:

Given the fact that "/users" is not a valid endpoint in itself, I find this kind of code line rather hard to read. Either provide fully valid relative URLs to your method, or hide the URL endpoint completely in the backen method.

File: src/views/Login.vue [hunk 2]
```diff
@@ -75,7 +76,7 @@ export default {
       console.log(localStorage.getItem('access_token'))
     }
   },
-  async created () {
+  async mounted () {
```
Comment (troeger) [inline src/views/Login.vue:79]: Code comments would be nice. What is "mounted"?

Comment (kat-hi) [reply src/views/Login.vue:79]: "created" happens before "mounted" and doesn't wait for async methods to complete. https://vuejs.org/v2/guide/instance.html#Lifecycle-Diagram

**Review by troeger — APPROVED** [review]
Awesome! Much better!

File: src/api/users.js (hunk 1 above)
Comment (kat-hi) [reply src/api/users.js]: I changed it

**Pull request merged**